include/event2/visibility.h

```diff
@@ -52,4 +52,11 @@
 
 #endif
 
+// For internal symbols referenced by event-extra
+#if defined(event_core_EXPORTS) || (defined(EVENT__NEED_DLLIMPORT) && defined(_MSC_VER) && !defined(EVENT_BUILDING_REGRESS_TEST))
+#define EVENT2_INTERNAL_EXPORT EVENT2_EXPORT_SYMBOL
```

Comment:
Let's let our compiler hide these functions and variables by default. Please revert back to just having extern N for internal variables needed in other code, this way visibility is hidden.